web-frontend/src/main.tsx

```diff
@@ -33,6 +34,7 @@ const router = createBrowserRouter(
       <Route path="error" element={<Error />} />
       <Route path="account-new" element={<AddAccount />} />
       <Route path="account-create-step1" element={<StepOne />} />
+      <Route path="account-create-step3" element={<StepThree />} />
```

Comment:
Why are you not applying that!? I did in the step-one, did in step-two and now in three.

```suggestion
      <Route path="account-create-step-three" element={<StepThree />} />
```

Reply:
My bad I honestly didn't see I'll create a mr to change it

Reply:
![Screenshot from 2023-05-16 10-35-26](https://github.com/massalabs/thyra-plugin-wallet/assets/90157528/91baa388-7434-405f-a6a0-da400ec875e7)

Sorry @mazmassa didn't see your comment because the branch was merged before I got around it... but it seems liek @Thykof knows and will make the changes... I'll change this current one :))